homeassistant/components/daikin/climate.py

```diff
@@ -118,8 +118,8 @@ async def async_setup_entry(
 
 
 def format_target_temperature(target_temperature):
-    """Format target temperature to be sent to the Daikin unit, taking care of keeping at most 1 decimal digit."""
-    return str(round(float(target_temperature), 1)).rstrip("0").rstrip(".")
+    """Format target temperature to be sent to the Daikin unit, rounding to nearest half degree."""
+    return str(round(float(target_temperature) * 2, 0) / 2).rstrip("0").rstrip(".")
```

Comment:
These formatting details are protocol details that should really be contained in the library. Hopefully the library can be improved.

Reply:
@vanackej I would be happy to introduce this in pydaikin, make a pr here: https://bitbucket.org/mustang51/pydaikin/src/master/

Reply:
Yes I could do that.
But are we sure first all Daikin wifi units behave the same way (allowing 0.5 temperature increment) ?
Don't you think releasing it there for a few time and wait for feedbacks before introducing it to the underlying library would be a good strategy ?

Reply:
Yes, you are right. Better to start here.